crates/bench/benches/subscription.rs

```diff
@@ -148,6 +151,57 @@ fn eval(c: &mut Criterion) {
         })
     });
 
+    //To profile this benchmark for 30s
+    //samply record -r 10000000 cargo bench --bench=subscription --profile=profiling -- select-protobuf --exact --profile-time=30
+    c.bench_function("select-protobuf", |b| {
+        // A passthru executed independently of the database.
+        let select_lhs = "select * from footprint";
+        let auth = AuthCtx::for_testing();
+        let tx = db.begin_tx();
+        let query_lhs = compile_read_only_query(&db, &tx, &auth, select_lhs).unwrap();
+        let query = ExecutionSet::from(query_lhs);
+        let out = query.eval_mem_table(&db, &tx).unwrap();
+
+        let out = DatabaseTableUpdate {
+            table_id: 0.into(),
+            table_name: "test".into(),
+            ops: out
+                .into_iter()
+                .flat_map(|x| x.data.into_iter().map(TableOp::insert))
+                .collect(),
+        };
```

Comment:
Right now `ExecutionSet::eval` returns a `ProtocolDatabaseUpdate`. Could you add a helper method that goes all the way to `DataMessage`?

What I'd like to benchmark is
```
ExecutionSet::eval
    -> ProtocolDatabaseUpdate (query execution)
    -> SubscriptionUpdateMessage (see ModuleSubscriptions::add_subscriber)
    -> DataMessage (see ClientConnectionSender::send_message)
```

That is, the full execution + the full serialization before we send over the wire.